web/app.js

```diff
@@ -2029,6 +2044,15 @@ const PDFViewerApplication = {
       _windowAbortController: { signal },
     } = this;
 
+    this._touchManager = new TouchManager({
```

Comment:
Should we also clear this field, by adding `this._touchManager = null;` at the end of the following method?
https://github.com/mozilla/pdf.js/blob/fbac4ab5f35dd4556069b69af51f78dbaaa62229/web/app.js#L2157-L2160